assets/js/components/activation/activation-app.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Activation App component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+
+/**
```

Comment:
I mean, wildly nitpicky, but aren't these to go above WP dependencies? I forget. 🤷‍♀

Reply:
Oh, also, these aren't really external? Whatevs 😆

Reply:
I don't think eslint cares about the order of the grouping, but apparently not.